doc/using_pytorch.rst

```diff
@@ -566,94 +566,127 @@ The function should return a byte array of data serialized to content_type.
-   result of invoking this function on the created endpoint name
--  ``env:`` Environment variables to run with
-   ``image`` when hosted in SageMaker.
--  ``name:`` The model name. If None, a default model name will be
-   selected on each ``deploy.``
--  ``entry_point:`` Path (absolute or relative) to the Python file
-   which should be executed as the entry point to model hosting.
--  ``source_dir:`` Optional. Path (absolute or relative) to a
-   directory with any other training source code dependencies including
-   the entry point file. Structure within this directory will be
-   preserved when training on SageMaker.
--  ``enable_cloudwatch_metrics:`` Optional. If true, training
-   and hosting containers will generate Cloudwatch metrics under the
-   AWS/SageMakerContainer namespace.
--  ``container_log_level:`` Log level to use within the container.
-   Valid values are defined in the Python logging module.
--  ``code_location:`` Optional. Name of the S3 bucket where your
-   custom code will be uploaded to. If not specified, will use the
-   SageMaker default bucket created by sagemaker.Session.
--  ``sagemaker_session:`` The SageMaker Session
-   object, used for SageMaker interaction
-
-Your model data must be a .tar.gz file in S3. SageMaker Training Job model data is saved to .tar.gz files in S3,
-however if you have local data you want to deploy, you can prepare the data yourself.
-
-Assuming you have a local directory containg your model data named "my_model" you can tar and gzip compress the file and
-upload to S3 using the following commands:
 
-::
+Now you can call the ``predict()`` method to get predictions from your deployed model.
 
-    tar -czf model.tar.gz my_model
-    aws s3 cp model.tar.gz s3://my-bucket/my-path/model.tar.gz
+***********************************************
+Attach an estimator to an existing training job
+***********************************************
+
+You can attach a PyTorch Estimator to an existing training job using the
+``attach`` method.
 
-This uploads the contents of my_model to a gzip compressed tar file to S3 in the bucket "my-bucket", with the key
-"my-path/model.tar.gz".
+.. code:: python
+
+    my_training_job_name = 'MyAwesomePyTorchTrainingJob'
+    pytorch_estimator = PyTorch.attach(my_training_job_name)
+
+After attaching, if the training job has finished with job status "Completed", it can be
+``deploy``\ ed to create a SageMaker Endpoint and return a
+``Predictor``. If the training job is in progress,
+attach will block and display log messages from the training job, until the training job completes.
 
-To run this command, you'll need the AWS CLI tool installed. Please refer to our `FAQ`_ for more information on
-installing this.
+The ``attach`` method accepts the following arguments:
 
-.. _FAQ: ../../../README.rst#faq
+-  ``training_job_name:`` The name of the training job to attach
+   to.
+-  ``sagemaker_session:`` The Session used
+   to interact with SageMaker
```

Comment:
I'd remove this and just link to https://sagemaker.readthedocs.io/en/stable/estimators.html#sagemaker.estimator.Framework.attach so that way we don't have to maintain the list of parameters in multiple places